app/api/answers/get_answer.go

```diff
@@ -10,38 +10,46 @@ import (
 
 // swagger:operation GET /answers/{answer_id} answers answerGet
 //
-//	---
-//	summary: Get an answer
-//	description: >
-//		Returns the answer identified by the given `{answer_id}`.
+//		---
+//		summary: Get an answer
+//		description: >
+//			Returns the answer identified by the given `{answer_id}`.
 //
-//		- If the user is a participant
-//			- (s)he should have at least 'content' access rights to the `answers.item_id` and
-//			- be a member of the `answers.participant_id` team or
-//				`answers.participant_id` should be equal to the user's self group.
+//			- If the user is a participant
+//				- (s)he (or one of his/her teams) should have at least 'content' access rights to the `answers.item_id` and
+//				- be a member of the `answers.participant_id` team or
+//				  `answers.participant_id` should be equal to the user's self group.
 //
-//		- If the user is an observer
-//			- (s)he should have `can_watch` >= 'answer' permission on the `answers.item_id` and
-//			- be a manager with `can_watch_members` of an ancestor of `answers.participant_id` group.
+//			- If the user is an observer/helper
```

Comment:
I think there is still a helper case missing: the user have can_watch>=answer permission on the item but no observation on the user, no validation. It does not match your criteria for being an observed or a helper.

Reply:
Fixed:
Now we allow viewing the answer if the thread exists AND the user has `can_watch>=answer` on the item. We do not check if either the user is a descendant of the helper_group_id or if the thread is expired in this case.

The purpose of checking if the thread exists in this case is not clear, but anyway...